<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeopleList</class>
 <widget class="QWidget" name="PeopleList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>524</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QFrame" name="toolBarFrame">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="margin">
         <number>2</number>
        </property>
        <item>
         <widget class="LineEditClear" name="filterLineEdit"/>
        </item>
        <item>
         <widget class="QToolButton" name="filterButton">
          <property name="icon">
           <iconset resource="../images.qrc">
            <normaloff>:/images/looknfeel.png</normaloff>:/images/looknfeel.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_NewId">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>Create new Identity</string>
          </property>
          <property name="text">
           <string>New ID</string>
          </property>
          <property name="icon">
           <iconset resource="../images.qrc">
            <normaloff>:/images/identity/identity_create_32.png</normaloff>:/images/identity/identity_create_32.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="filterComboBox"/>
     </item>
     <item>
      <widget class="RSTreeWidget" name="idTreeWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="headerStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Person name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Identity ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Activity</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="editIdentity">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/edit_16.png</normaloff>:/images/edit_16.png</iconset>
   </property>
   <property name="text">
    <string>Edit identity</string>
   </property>
   <property name="toolTip">
    <string>Edit identity</string>
   </property>
  </action>
  <action name="removeIdentity">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete identity</string>
   </property>
  </action>
  <action name="chatIdentity">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/toaster/chat.png</normaloff>:/images/toaster/chat.png</iconset>
   </property>
   <property name="text">
    <string>Chat with this peer</string>
   </property>
   <property name="toolTip">
    <string>Launches a distant chat with this peer</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RSTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>gui/common/RSTreeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>LineEditClear</class>
   <extends>QLineEdit</extends>
   <header location="global">gui/common/LineEditClear.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>idTreeWidget</tabstop>
 </tabstops>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
